src/main/python/hydra_deploy/mesos_marathon_setup.py

```diff
@@ -194,8 +194,9 @@ def setup(step, config):
     parser.add_argument('--config_file', '-f', type=str,
                         default=os.getcwd() + "/setup_config.ini",
                         help='Setup configuration file. File holds your gcloud confiuration settings. It also holds '
-                             'values of number of master/slave nodes.A sample configuration file can be found at '
-                             'hydra-deploy/src/main/python/gcloud/setup_config.ini. ')
+                             'values of number of master/slave nodes. A sample configuration file can be found at '
+                             'hydra-deploy/src/main/python/gcloud/setup_config.ini (This is also default conf file). ')
+
```

Comment:
The master script currently just exists at the end of the install, 
it should have a nice success message with the list of instances installed and there ip addresses.